lifecycle/export.go

```diff
@@ -0,0 +1,133 @@
+	}
+	// Start the Export process and wait for completion
+	log.Info("Exporting Release Bundle archive...")
+	exportResponse, err := servicesManager.ExportReleaseBundle(rbDetails, rbe.modifications, queryParams)
+	if err != nil {
+		return errorutils.CheckErrorf("Failed exporting release bundle, error: '%s'", err.Error())
+	}
+	// Download the exported bundle
+	log.Debug("Downloading the exported bundle...")
+	_, failed, err := rbe.downloadReleaseBundle(exportResponse, rbe.downloadConfigurations)
+	if err != nil || failed > 0 {
+		return
+	}
+	log.Info("Successfully Downloaded Release Bundle archive")
+	return
+}
+
+// Download the exported release bundle using artifactory service manager
+func (rbe *ReleaseBundleExportCommand) downloadReleaseBundle(exportResponse services.ReleaseBundleExportedStatusResponse, downloadConfiguration artUtils.DownloadConfiguration) (downloaded int, failed int, err error) {
+	downloadParams := artServices.DownloadParams{
+		CommonParams: &utils.CommonParams{
+			Pattern: strings.TrimPrefix(exportResponse.RelativeUrl, "/"),
+			Target:  rbe.targetPath,
+		},
+		MinSplitSize: downloadConfiguration.MinSplitSize,
+		SplitCount:   downloadConfiguration.SplitCount,
+	}
+	artifactoryServiceManager, err := createArtifactoryServiceManager(rbe.serverDetails)
+	if err != nil {
+		return
+	}
+	return artifactoryServiceManager.DownloadFiles(downloadParams)
+
+}
+func (rbe *ReleaseBundleExportCommand) ServerDetails() (*config.ServerDetails, error) {
+	return rbe.serverDetails, nil
+}
+
+func (rbe *ReleaseBundleExportCommand) CommandName() string {
+	return "rb_export"
+}
+
+func NewReleaseBundleExportCommand() *ReleaseBundleExportCommand {
+	return &ReleaseBundleExportCommand{}
+}
+func (rbe *ReleaseBundleExportCommand) SetServerDetails(serverDetails *config.ServerDetails) *ReleaseBundleExportCommand {
+	rbe.serverDetails = serverDetails
+	rbe.releaseBundleCmd.serverDetails = serverDetails
+	return rbe
+}
+
+func (rbe *ReleaseBundleExportCommand) SetReleaseBundleExportModifications(modifications services.Modifications) *ReleaseBundleExportCommand {
+	rbe.modifications = modifications
+	return rbe
+}
+func (rbe *ReleaseBundleExportCommand) SetReleaseBundleName(releaseBundleName string) *ReleaseBundleExportCommand {
+	rbe.releaseBundleName = releaseBundleName
+	return rbe
+}
+
+func (rbe *ReleaseBundleExportCommand) SetReleaseBundleVersion(releaseBundleVersion string) *ReleaseBundleExportCommand {
+	rbe.releaseBundleVersion = releaseBundleVersion
+	return rbe
+}
+
+func (rbe *ReleaseBundleExportCommand) SetProject(project string) *ReleaseBundleExportCommand {
+	rbe.rbProjectKey = project
+	return rbe
+}
+
+func (rbe *ReleaseBundleExportCommand) SetDownloadConfiguration(downloadConfig artUtils.DownloadConfiguration) *ReleaseBundleExportCommand {
+	rbe.downloadConfigurations = downloadConfig
+	return rbe
+}
+
+func (rbe *ReleaseBundleExportCommand) SetTargetPath(target string) *ReleaseBundleExportCommand {
```

Comment:
We can allow target path to include a file name to, not necessarily a directory. Lets mimic the behavior as in the download command.